data/blog/app-feedback-program.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Electron App Feedback Program
+author: sofianguy
+date: '2018-10-04'
+---
+
+Electron is working on building a streamlined release process and having faster releases. To help with that, we started the App Feedback Program for large-scale Electron apps to test our beta releases and report app-specific issues to the Electron team. We use this program to help us prioritize work and get applications upgraded to the next stable release as soon as possible.
+
```

Comment:
Somewhere in here, usually after the first paragraph, you should add a line with `---` to denote what content shows up as the summary on the http://electronjs.org/blog page.

Reply:
See other blog posts for reference.

Reply:
Ohhh that's how that works. Ok thanks! 👍